DashAI/back/dataloaders/classes/dataloader.py

```diff
@@ -0,0 +1,70 @@
+from abc import abstractmethod
+
+from datasets import ClassLabel
+
+from DashAI.back.config_object import ConfigObject
+
+
+class DataLoader(ConfigObject):
+    """
+    Abstract class with base methods for all data loaders
+    """
+
+    @abstractmethod
+    def load_data(self, dataset_path, url=None):
+        pass
+
+    @abstractmethod
+    def set_task_format(self, task):
+        pass
+
+    def split_dataset(self, dataset, params, class_column):
+        """
+        Split the dataset in train, test and validation data
+        """
+        stratify_column = None
+        if params["stratify"]:
+            stratify_column = class_column
```

Comment:
- I considered that if it's not a classification problem, the 'stratify' parameter would be false by default, because in the front end this option should not be shown to the user, so takes the default value of pydantic model DatasetParams, which is not really clear at all, so I will define this default value by setting the explicit parameters as you said, instead of receiving the dictionary. For that if you pass a class column, nothing would be done with it.
- Indeed, it is necessary to check if it exists, but now I will change that to the explicits parameters.
- Before calling this function, it is validated that the format of the dataset is compatible with the task and there it is verified that the dataset contains the column that was indicated as a class column, which was not yet included in this branch, but I will update it and leave a comment about this.